sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample11_SettingsSnapshot.md

```diff
@@ -0,0 +1,127 @@
+# Create, Retrieve and Update status of a Configuration Settings Snapshot
+
+Azure Configuration Settings Snapshot allow users to create a point-in-time snapshot of their configuration store, providing them with enhanced flexibility and control over their configuration settings. This feature enables users to create backups and easily restore their configurations to a previous state when needed. Users can create immutable, composed views of their configuration store that they can release with their application. The immutability of these snapshots ensures that users can confidently rollback to a last-known-good configuration in the event of an issue with newer configuration. This sample illustrates how to create, retrieve, and modify the status of a `ConfigurationSettingSnapshot`.
```

Comment:
Can we update this paragraph with the same text that you decide to use for the README?  I made suggestions there.